radix-engine-tests/tests/package_loader.rs

```diff
@@ -0,0 +1,44 @@
+#[cfg(feature = "compile-blueprints-at-build-time")]
+mod package_loader {
+    use radix_engine_common::prelude::*;
+    use radix_engine_queries::typed_substate_layout::*;
+    use std::collections::HashMap;
+
+    const PACKAGES_BINARY: &[u8] =
+        include_bytes!(concat!(env!("OUT_DIR"), "/compiled_packages.bin"));
+
+    lazy_static::lazy_static! {
+        static ref PACKAGES: HashMap<String, (Vec<u8>, PackageDefinition)> = {
+            scrypto_decode(PACKAGES_BINARY).unwrap()
+        };
+    }
+
+    pub struct PackageLoader;
+    impl PackageLoader {
+        pub fn get(name: &str) -> (Vec<u8>, PackageDefinition) {
+            if let Some(rtn) = PACKAGES.get(name) {
+                rtn.clone()
+            } else {
+                panic!("Package \"{}\" not found. Are you sure that this package is: a) in the blueprints folder, b) that this is the same as the package name in the Cargo.toml file?", name)
```

Comment:
Just wondering if it make sense to compile the package here instead of panicking.
Not really sure.
On the one hand we would just go further without stopping.
On the other we could hide some problem by silently rebuilding the missing package.

Reply:
I personally think that it might be better to have it fail instead of succeed since the main objective of this is to eliminate any contention that might exist on the files. So, this would alert the test author and they'd make the changes that need to be made as per the provided suggestions.